util/logging.go

```diff
@@ -42,6 +53,16 @@ var (
 	// All go-vcloud director logging goes through this logger
 	GovcdLogger *log.Logger
 
+	// HTTP operation stack
+	GovcdHttpStack FixedSizeStack
+
+	// HTTP operation stack size
+	// 4 operations == two requests and two responses
+	GovcdHttpStackSize int = 4
```

Comment:
Is there a way users can set this

Reply:
Yes. It's public interface. We may add these parameters to the test configuration, to be used instead of regular logging.